packages/block-editor/src/components/inserter/index.native.js

```diff
@@ -32,27 +33,64 @@ import { store as blockEditorStore } from '../../store';
 
 const VOICE_OVER_ANNOUNCEMENT_DELAY = 1000;
 
-const defaultRenderToggle = ( { onToggle, disabled, style, onLongPress } ) => (
-	<ToolbarButton
-		title={ _x( 'Add block', 'Generic label for block inserter button' ) }
-		icon={
+const defaultRenderToggle = ( {
+	onToggle,
+	disabled,
+	style,
+	onLongPress,
+	isDefaultView,
+} ) => {
+	// The "default view" refers to the editor's appearance when no blocks
```

Comment:
Interested in thoughts around whether this comment is justified. It felt like the reasoning for separate "default view" variables might not be clear when I was working on this, but perhaps I'm wrong on that.

Reply:
I like it. "Default view" is a bit vague , which is fine but the comment helps shore up the meaning.